includes/rest/class-server.php

```diff
@@ -72,9 +72,9 @@ public static function authorize_activitypub_requests( $response, $handler, $req
 
 		// check if it is an activitypub request and exclude webfinger and nodeinfo endpoints
 		if (
-			! str_starts_with( $route, '/' . ACTIVITYPUB_REST_NAMESPACE ) ||
-			str_starts_with( $route, '/' . \trailingslashit( ACTIVITYPUB_REST_NAMESPACE ) . 'webfinger' ) ||
-			str_starts_with( $route, '/' . \trailingslashit( ACTIVITYPUB_REST_NAMESPACE ) . 'nodeinfo' )
+			0 !== strpos( $route, '/' . ACTIVITYPUB_REST_NAMESPACE ) ||
+			0 === strpos( $route, '/' . \trailingslashit( ACTIVITYPUB_REST_NAMESPACE ) . 'webfinger' ) ||
+			0 === strpos( $route, '/' . \trailingslashit( ACTIVITYPUB_REST_NAMESPACE ) . 'nodeinfo' )
```

Comment:
WordPress includes a polyfill for `str_starts_with`, so I don't think we need to make that change here:
https://github.com/WordPress/WordPress/blob/65a442fbcd370e395be94fe0aa3e824225f7edc5/wp-includes/compat.php#L458

Reply:
oh nice, wasn't aware of that.

Reply:
You might need to put it in the global scope, though, by adding a `\`